contracts/test/core/invoke/Issuance.t.sol

```diff
@@ -7,39 +7,40 @@ import {SCALAR, fmul} from "src/lib/FixedPoint.sol";
 import {IVault} from "src/interfaces/IVault.sol";
 
 contract IssuanceTest is StatefulTest {
-    function testGoToZero() public {
```

Comment:
commented these out for now because it wont go to zero because fees are minted before burn

Reply:
the "go to zero" is actually misleading here, these tests actually now check whether there's dust left over or not.